plugins/notify.py

```diff
@@ -15,26 +15,28 @@ def send_notification(self, build_id, text):
             for room in notification_list:
                 self.say("{} BuildID: {}, Message: {}".format(' '.join('@'+user for user in notification_list.get(room, [])), build_id, text), room=self.get_room_from_name_or_id(room), notify=True) 
 
-    @respond_to("^subscribe (@?)(?P<user>\S+) to (?P<build_id>\S+)")
+    @respond_to("^subscribe (@?)(?P<users>(\S+ )+)to(?P<build_ids>( \S+)+)")
     def subscribe(self, message, user, build_id):
```

Comment:
`users` gets kwarg mapped to the function params.  s/user/users/ in the param names.